packages/vcf-lv2/PKGBUILD

```diff
@@ -0,0 +1,36 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Florian Hülsmann <fh@cbix.de>
+
+pkgname=vcf-lv2
+pkgver=0.0.2
+pkgrel=1
+pkgdesc='VCF LV2 plugins'
+arch=(x86_64 aarch64)
+url='https://github.com/linuxmao-org/vcf-lv2'
+license=(GPL2)
+groups=(lv2-plugins pro-audio)
+depends=(glibc)
+makedepends=(lv2)
+checkdepends=(lv2lint)
+optdepends=('lv2-host: for running the LV2 plugin')
+source=("$pkgname-$pkgver.tar.gz::https://github.com/linuxmao-org/$pkgname/archive/refs/tags/v$pkgver.tar.gz")
+sha256sums=('4dc4c7eb3a81cfcbffaabe2c0bac229e6806f984140c6a0f4d0e74c988600bd7')
+
+build() {
+  cd $pkgname-$pkgver
+  make
+}
+
+check() {
+  cd $pkgname-$pkgver
+  _lv2_uri='http://jwm-art.net/lv2/vcf'
+  for _plugin in plugins/*.lv2; do
+    lv2lint -Mpack -I $_plugin $_lv2_uri/${_plugin:8:-8}{,_cv} ||
```

Comment:
This outputs these errors for each plugin:

```
    [FAIL]  Plugin Version Minor
              lv2:minorVersion not found
              seeAlso: <http://lv2plug.in/ns/lv2core#minorVersion>
    [FAIL]  Plugin Version Micro
              lv2:microVersion not found
              seeAlso: <http://lv2plug.in/ns/lv2core#microVersion>
    [FAIL]  Plugin Features
              lv2:[optional|required]Feature <http://lv2plug.in/ns/lv2core#hardRtCapable> not valid
              seeAlso: <http://lv2plug.in/ns/lv2core#Feature>
```

You could ignore these with the `-t ` option, then you don't need to ignore the exit code:

```
  for _plugin in plugins/*.lv2; do
    lv2lint -Mpack -t "Plugin Version *" -t "Plugin Features" \
      -I $_plugin $_lv2_uri/${_plugin:8:-8}{,_cv}
  done
```

Reply:
0875dcc9b5c991a67070b9950ad808f258035b5c